src/commands/action-flows/module.ts

```diff
@@ -4,12 +4,75 @@
 
 import { Configurator, IModule } from "../../core/command/module-handler";
 import { Context } from "../../core/command/cli-context";
+import { Command, OptionValues } from "commander";
+import { ActionFlowCommandService } from "./action-flow/action-flow-command.service";
+import { SkillCommandService } from "./skill/skill-command.service";
 
 class Module extends IModule {
 
     register(context: Context, configurator: Configurator) {
+        const analyzeCommand = configurator.command("analyze");
+        analyzeCommand.command("action-flows")
+            .description("Analyze Action Flows dependencies for a certain package")
+            .option("-p, --profile <profile>", "Profile which you want to use to analyze Action Flows")
```

Comment:
Isn't that (--profile) now at the core level, so could be removed from any module specific commands?